coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 99.64
+    "covered_percent": 0.0
```

Comment:
I can assign someone else if you don't want to, but it looks like you started working on it.